helm/inji-verify/values.yaml

```diff
@@ -415,7 +415,12 @@ metrics:
     rules: []
 
 ## The url below is to access esignet API services.
-## CAUTION: Domain name to access injiweb UI must be secure/internal/over vpn.
+## CAUTION: Domain name to access injiverify UI must be secure/internal/over vpn.
+## Slash is important!
+inji_verify:
+  inji_verify_port: '3004'
```

Comment:
instead of inji_verify_port and inji_verify_host lets have it as host and port only

Reply:
Updated as per comments .